code/drasil-example/glassbr/lib/Drasil/GlassBR/Unitals.hs

```diff
@@ -106,7 +106,7 @@ standOffDist = uq (constrained' (uc sD (variable "SD") Real metre)
 
 nomThick = cuc "nomThick" 
   (nounPhraseSent $ S "nominal thickness" +:+ displayDblConstrntsAsSet 
-    nomThick nominalThicknesses)
+    (mkQuant "nomThick" (nounPhraseSent $ S "nominal thickness") lT Rational Nothing Nothing) nominalThicknesses)
```

Comment:
Seeing this added here, I would imagine it would be removed from somewhere else, too, no? Does this result in code duplication?

Reply:
I'm not sure what you are referring to, could you be more specific?

Reply:
I _think_ the underlying question is "is this information written anywhere else in our code base"?

A similar thought I had: the duplication of `S "nominal thickness"` and the use of the `Nothing`s seems to suggest that this chunk could be built with a different constructor than `cuc` that builds it more "logically", but I'm not sure what other constructors are available.…

Reply:
Why would we not use cuc? The `Nothing`s and the `S "nominal thickness"` are used in the `mkQuant` constructor. The `displayDblConstrntsAsSet` function uses this to make a sentence for the `cuc` constructor.

Reply:
What I mean by this is that we are duplicating the arguments `"nomThick"`, `nounPhraseSent $ S "nominal thickness" +:+ x`, `lT`, and `Rational`, which seems redundant. Are these defined anywhere else in the code that we could borrow from instead of redefining it (there may not be)? Looking at the constructors for `ConstrainedChunk`, it looks like they all require some "base" "chunk" to build off of (which is one argument for moving away from our "nested" infrastructure), so we might be stuck using `cuc`.

While investigating this, I noticed that the reason we have this "circular definition" is because we seem to be attempting to define it in terms of itself, instead of just actually giving it a definition, which prompted [https://github.com/JacquesCarette/Drasil/issues/3815#issuecomment-2197788525](a potential agenda item for our next meeting). 👀

Reply:
I agree that this "smells bad". Feels like the wrong constructor is being used.